tests/ut/language/parser/test_error_cases.py

```
@@ -122,6 +122,26 @@ def bad_target(n: pl.Tensor[[1], pl.INT32]) -> pl.Tensor[[1], pl.INT32]:
+class ChunkedLoopProgram:
+    @pl.function(type=pl.FunctionType.Orchestration)
+    def main(
+        self,
+        x: pl.Tensor[[16, 4], pl.FP32],
+        seq_lens: pl.Tensor[[16], pl.INT32],
+    ) -> pl.Tensor[[16, 4], pl.FP32]:
+        for b in pl.parallel(0, 16, 1, chunk=4):
+            _ctx_len = pl.tensor.read(seq_lens, [b])
+        return x
+"""
+        with pytest.raises(ParserSyntaxError, match="only valid inside with pl.auto_incore"):
```

Ruff (0.15.6)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)